src/library/masq.js

```diff
@@ -138,7 +141,7 @@ class Masq {
     await db.putAsync('/profile/protectedMK', protectedMK)
     // We do not use this.encryptAndPut method because the master key
     // is not stored in this.profileDB (only set in openProfile)
-    await put(db, this.masterKey, '/profile', privateProfile)
+    await put(db, this.masterKey, '/profile', privateProfile, nonce)
```

Comment:
I think @sujeebant already made a comment about this, but what about passing the nonce as third parameter instead ?

Reply:
yes, I had not yet updated masq-app. I do it right now.

Reply:
done